content/en/welcome/_index.md

```diff
@@ -33,7 +33,7 @@ contributor resources, Kubernetes SIGs, and more.
 <h2>Get Involved:</h2>
 
 The first step is to check out the <a href="/docs/guide/">contributor guide</a>.
-All contributors must abide by the <a href="/resources/code-of-conduct/">CNCF Code of Conduct</a>
+All contributors must abide by the <a href="/community/code-of-conduct/">CNCF Code of Conduct</a>
```

Comment:
This link is broken. Better use - https://github.com/kubernetes/community/blob/master/code-of-conduct.md

Reply:
https://www.kubernetes.dev/community/code-of-conduct/ in turn shows some content and also links to  https://git.k8s.io/community/code-of-conduct.md / https://github.com/kubernetes/community/blob/master/code-of-conduct.md

Reply:
@karuppiah7890 is correct here, we want to link to our version 👍 
/lgtm
/approve